### PR TITLE
Elena/login

### PR DESCRIPTION
**Pull request by ElenaKrumina**
View for login page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import React, { Component } from "react";
 import "./App.css";
+import Login  from  "./components/Login/Login"
 
 class App extends Component {
   render() {
-    return <div className="App">React App</div>;
+    return <div className="App">
```
Comment (rokija) [inline src/App.js:7]: please format this wrapping in braces () and using indentation properly. (code style)

File: src/components/Login/Login.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-image: url("../../assets/photo-pic.jpg");
+}
+.Login__content {
+    width: 35%;
+    background-color: rgb(202, 203, 195);
+    padding:61px;
```
Comment (rokija) [inline src/components/Login/Login.css]: put space here between padding: and 61px (code style),
also why 61px not 60px?

File: src/components/Login/Login.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    display: flex;
+    justify-content: center;
+}
+.Login-button {
+    margin-right: 15px;
+}
+ .img {
+      width: 500%; 
+      height: 500%;
+
```
Comment (rokija) [inline src/components/Login/Login.css]: remove empty line (code style)

File: src/components/Login/Login.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+.buttons {
+    display: flex;
+    justify-content: center;
+}
+.Login-button {
+    margin-right: 15px;
+}
+ .img {
+      width: 500%;
```
Comment (rokija) [inline src/components/Login/Login.css]: why 500%?

File: src/components/Login/Login.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+            <Button className="Login-button" color="dark">
+              Login
+            </Button>
+
+            <Button color="dark">
+              Register
+            </Button>
+          
+            </div>
+
```
Comment (rokija) [inline src/components/Login/Login.js]: remove empty line (code style)

File: src/components/Login/Login.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+            </Button>
+
+            <Button color="dark">
+              Register
+            </Button>
+          
+            </div>
+       
+          </form>
+
```
Comment (rokija) [inline src/components/Login/Login.js]: remove empty line (code style)

File: src/components/Login/Login.js [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+         </div>
+      </div>
+    );
+  }
+}
+
+export default Login;
```
Comment (rokija) [inline src/components/Login/Login.js]: add new line after this line

**Review by rokija — CHANGES_REQUESTED** [review]
please fix styles for mobile view.

**Review by rokija — APPROVED** [review]

**Review by Kurkos — APPROVED** [review]
All is fine!

**Pull request merged**